content/gjtempleton/drone-pypi/index.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2018-02-11T00:00:00+00:00
+title: Pypi
+author: gjtempleton
+tags: [ python, pypi ]
+repo: gjtempleton/drone-pypi
+logo: pypi.svg
+image: gjtempleton/drone-pypi
+---
+
+This plugin allows you to publish Python pypi packages as part of your pipeline.
+
+Basic example:
+
+```yaml
+pipeline:
+    pypi_publish:
```

Comment:
Can you make the yaml indents to 2 spaces please?

Reply:
Sure, done, thanks for the review!